README.md

```diff
@@ -201,3 +201,28 @@ The general `toc-entry` is applied to all `li` elements in the `ul.section-nav`.
 
 Depending on the heading level each specific entry refers to, it has a second CSS class `toc-XX`, where `XX` is the HTML heading tag name. For example, the TOC entry linking to a heading `<h1>...</h1>` (a single
 `#` in Markdown) will get the CSS class `toc-h1`.
+
+#### Anchor injection
+
+Anchor injection can be customized with a liquid template:
+
+```yml
+toc:
+  injection_template: 'path/to/template.html'
+```
+
+The template is supplied with the original text content (`heading_text`) and id (`heading_id`) for each heading element:
+
+```liquid
+<!-- Append an anchor pilcrow after the heading text, Sphinx style: -->
+{{ heading_text }} <a class="anchor" aria-hidden="true" href="{{ page.url }}{{ heading_id }}">¶</a>
```

Comment:
I'm wondering why you need to configure `page.url`. 

I think anchor's `href` should be `#heading-title`, for instance. 🤔

Reply:
In this example, the idea is for the anchor to provide an absolute reference so right click -> copy link give something which can be pasted anywhere. Of course, the goal is for customization; you could of course just use `#{{ heading_id }}` to get that behavior, if that is what one wishes.